automatic/1password8/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,14 @@
+$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName    = $env:ChocolateyPackageName
+  fileType       = 'msi'
+  url            = ''
+  softwareName   = '1Password*'
+  checksum       = ''
+  checksumType   = 'sha256'
+  silentArgs     = "/qn /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+  validExitCodes = @(0)
```

Comment:
Reminder to myself, the exit codes need to be updated to confirm to our `Pending` guidelines.